src/index.js

```diff
@@ -0,0 +1,42 @@
+import './style.css';
+
+const list = [
+  {
+    description: 'complete course',
+    index: 0,
+    complete: true,
+  },
+  {
+    description: 'Meet rina',
+    index: 1,
+    complete: true,
+  },
+  {
+    description: 'buy eggs',
+    index: 2,
+    complete: false,
+  },
+  {
+    description: 'go to gym',
+    index: 3,
+    complete: false,
+  },
+];
+const ul = document.querySelector('#myUL');
+// Create element
+const generate = (descValue) => {
+  const li = document.createElement('li');
+  const check = document.createElement('input');
+  const label = document.createElement('label');
+  check.type = 'checkbox';
+  ul.appendChild(li);
+  li.innerHTML = descValue;
+  li.appendChild(label);
+  label.appendChild(check);
+  check.setAttribute('class', 'check');
+  check.setAttribute('class', 'check');
+  label.setAttribute('class', 'label');
+};
+list.forEach((p) => {
```

Comment:
- [ ] Yout todo app should look like the provided design, kindly adjust your design so it looks identical to the provided one

![Screenshot_20220928_195248](https://user-images.githubusercontent.com/25227022/192865335-b291a129-9e07-4cba-9789-7168aaefc5d7.png)